modules/model/shared/src/main/scala/observe/model/package.scala

```diff
@@ -28,24 +25,12 @@ val CalibrationQueueName: String = "Calibration Queue"
 val CalibrationQueueId: QueueId  =
   QueueId(UUID.fromString("7156fa7e-48a6-49d1-a267-dbf3bbaa7577"))
 
-given Eq[TimeUnit] =
-  Eq.by(_.symbol)
-
-given Eq[Time] =
-  Eq.by(_.toMilliseconds)
-
 extension (i: Instrument)
   def hasOI: Boolean = i match
 //      case Instrument.F2    => true
     case Instrument.GmosSouth => true
     case Instrument.GmosNorth => true
     case _                    => false
-//      case Instrument.Nifs  => true
-//      case Instrument.Niri  => true
-//      case Instrument.Gnirs => true
-//      case Instrument.Gsaoi => false
-//      case Instrument.Gpi   => true
-//      case Instrument.Ghost => false
```

Comment:
I left those there, to restore them later as we implement support for more instruments.